pkg/golinters/goanalysis/runner.go

```diff
@@ -935,7 +935,8 @@ func sizeOfReflectValueTreeBytes(rv reflect.Value, visitedPtrs map[uintptr]struc
 		return rv.Len()
 	case reflect.Int, reflect.Int8, reflect.Int16, reflect.Int32, reflect.Int64,
 		reflect.Uint, reflect.Uint8, reflect.Uint16, reflect.Uint32, reflect.Uint64,
-		reflect.Uintptr, reflect.Bool, reflect.Float32, reflect.Float64, reflect.UnsafePointer:
+		reflect.Uintptr, reflect.Bool, reflect.Float32, reflect.Float64,
+		reflect.Complex64, reflect.Complex128, reflect.Func, reflect.UnsafePointer:
```

Comment:
These missing cases
```
reflect.Complex64, reflect.Complex128, reflect.Func
```
were reported by the exhaustive linter when I temporarily enabled it for a run.